image.c

```diff
@@ -332,6 +548,10 @@ bool img_load(img_t *img, const fileinfo_t *file)
 #if HAVE_GIFLIB
 		if (STREQ(fmt, "gif"))
 			img_load_gif(img, file);
+#endif
+#if HAVE_LIBWEBP
+		if (STREQ(fmt, "webp"))
+			img_load_webp(img, file);
```

Comment:
This seems to to be the only place where this function is called. And the return value isn't even used.

Reply:
The function is implemented following `img_load_gif()`.

I don't think only calling it in one place is a big issue, as having that logic in its own function makes sense considering that `img_load()` is high-level and abstracted from the details of any image format.

I agree that the return value not being used is a problem, but the same problem exists for `img_load_gif()`, so I think the problem lies witht the implementation of the `img_load()` function. It shouldn't ignore the return values of `img_load_gif()` or `img_load_webp()`, and should instead check if they returned true before continuing with the rest of the function, bailing out and returning false if not.

Reply:
> I don't think only calling it in one place is a big issue

Apologies for being misleading. My point was that since the two functions had different return types, I thought that they couldn't be merged, but then I see that this function's return value isn't even used, so I'm thinking, "I guess it can be merged (?)"